### PR TITLE
`generic-autobumper`: add option to append to PR body

### PR DESCRIPTION
**Pull request by smg247**
OpenShift would like to automatically command our `pj-rehearse` external prow plugin upon creation of these autobump PRs. In order to do that we need to be able to configure additional text in the PR body.

### CODE REVIEW

**Comment by smg247** [comment]
@chaodaiG do you mind taking a look at this one?

**Review by chaodaiG — APPROVED** [review]
/hold
I'm a little surprised that there is no unit test for this function, would be nice to have. However I'm not gonna request it just because you touched this file, feel free to unhold

**Comment by smg247** [comment]
/hold cancel
I do think a unit test would be nice to have. I am going to get this merged to unblock the rest of my feature, but I will come back and add one soon.

**Pull request merged**
